risedev.yml

```diff
@@ -1446,3 +1480,32 @@ template:
     # If `user-managed` is true, user is responsible for starting the service
     # to serve at the above address and port in any way they see fit.
     user-managed: false
+
+  # PostgreSQL service backed by docker.
+  postgres:
+    # Id to be picked-up by services
+    id: postgres-${port}
+
+    # address of pg
+    address: "127.0.0.1"
+
+    # listen port of pg
+    port: 8432
+
+    # Note:
+    # - This will be used to initialize the PostgreSQL instance if it's fresh.
+    # - In user-managed mode, these configs are not validated by risedev.
+    #   They are passed as-is to risedev-env default user for PostgreSQL operations.
```

Comment:
Should also add the part for writing postgres envs that can be read by `psql` here:

https://github.com/risingwavelabs/risingwave/blob/c52cb9c5dd8f04931b4da6a235509cf855c13d9f/src/risedevtool/src/risedev_env.rs#L80-L93

Reply:
> can be read by `psql` here:

To clarify, the `psql` is that used in `slt` files, and loaded when using `risedev slt`. It's mainly for pg as a source, not for meta backend.

e.g., 

https://github.com/risingwavelabs/risingwave/blob/07fc213b160bf4d092754eda5bc8e90fb69113c1/e2e_test/source_inline/cdc/mysql/mysql_create_drop.slt#L8-L10